posthog/queries/paths.py

```diff
@@ -14,6 +14,9 @@
 
 
 class Paths(BaseQuery):
+    def __init__(self, *args, **kwargs) -> None:
```

Comment:
There's also a divergence from how old Paths + Postgres Paths used to work, this ensures at least tests for all 3 continue passing.